.github/workflows/version-update.yml

```diff
@@ -0,0 +1,40 @@
+name: Update package version for release & hotfix branches
+
+on:
+  create:
+    branches: [release/*, hotfix/*]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+      - name: update package version
+        id: vars
+        run: |
+          BRANCH_NAME="${GITHUB_REF#refs/heads/}"
+          PACKAGE_VERSION="${GITHUB_REF##*/}"
+          echo ::set-output name=branch::${BRANCH_NAME}
+          echo ::set-output name=version::${PACKAGE_VERSION}
+          if [[ $PACKAGE_VERSION =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]
+          then
+            git config user.name 'github-actions[bot]'
+            git config user.email 'github-actions[bot]@users.noreply.github.com'
+            npm version ${PACKAGE_VERSION}
+            git push -u origin HEAD:"dev/update-version-${PACKAGE_VERSION}"
+          else
+            echo "Branch name ${BRANCH_NAME} does not have the correct format with package version."
+            exit 1
+          fi
+      - uses: repo-sync/pull-request@v2
+        with:
+          source_branch: "dev/update-version-${{ steps.vars.outputs.version }}"
+          destination_branch: "${{ steps.vars.outputs.branch }}"
+          pr_title: "Update Package Version to ${{ steps.vars.outputs.version }}"
+          pr_body: "*An automated PR*"
```

Comment:
nit: Can we add a little more to the pr_body such as `An automated PR which updates the version`?

Reply:
sure thing